pootle/apps/pootle_fs/resources.py

```diff
@@ -47,3 +53,111 @@ def unsynced(self):
     @property
     def trackable_stores(self):
         return self.stores.exclude(obsolete=True).filter(fs__isnull=True)
+
+
+class FSProjectStateResources(object):
+    """Wrap FSPlugin and cache available resources
+    Accepts `pootle_path` and `fs_path` glob arguments.
+    If present all resources are filtered accordingly.
+    """
+
+    def __init__(self, context, pootle_path=None, fs_path=None):
+        self.context = context
+        self.pootle_path = pootle_path
+        self.fs_path = fs_path
+
+    def match_fs_path(self, path):
+        """Filter fs_paths using file glob"""
```

Comment:
hmm well it does filter in the sense that it only returns if matched (or no match required)